src/plugins/navigation/mutations/publishNavigationChanges.js

```diff
@@ -6,20 +6,20 @@ import getNavigationTreeItemIds from "../util/getNavigationTreeItemIds.js";
  * @method publishNavigationChanges
  * @summary Publishes changes for a navigation tree and its items
  * @param {Object} context An object containing the per-request state
- * @param {String} _id _id of navigation tree to publish
+ * @param {String} input input of navigation tree to publish
+ * @param {String} shopId shopId of navigation tree to publish
  * @returns {Promise<Object>} Updated navigation tree
  */
-export default async function publishNavigationChanges(context, _id) {
+export default async function publishNavigationChanges(context, input) {
```

Comment:
This is called internally by `src/plugins/navigation/util/createDefaultNavigationTreeForShop.js`. Need to update the function signature there.